config/v201/CMakeLists.txt

```diff
@@ -26,15 +35,6 @@ install(
 if (LIBOCPP_INSTALL_DEVICE_MODEL_DATABASE)
      set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/config.json)
```

Comment:
```suggestion
```
This can be removed now since we dont need cmake to run anymore if the config changes